float64 x_orient_in
float64 y_orient_in
float64 z_orient_in
float64 w_orient_in
float64 x_angularvel_in
float64 y_angularvel_in
float64 z_angularvel_in
float64 x_accel_in
float64 y_accel_in
float64 z_accel_in
---
float64 x_orient_out
float64 x_orient_experienced
float64 y_orient_out
float64 y_orient_experienced
float64 z_orient_out
float64 z_orient_experienced
float64 w_orient_out
float64 w_orient_experienced
float64 x_angularvel_out
float64 x_angularvel_experienced
float64 y_angularvel_out
float64 y_angularvel_experienced
float64 z_angularvel_out
float64 z_angularvel_experienced
float64 x_accel_out
float64 x_accel_experienced
float64 y_accel_out
float64 y_accel_experienced
float64 z_accel_out
float64 z_accel_experienced
bool success
